apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -25,28 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	);
 
 	return (
-		<>
-			<h3>Sign Up</h3>
-			<form onSubmit={handleFormSubmit}>
+		<AuthLayout>
+			<h3 className={styles["title"]}>Create an account</h3>
+			<form className={styles["form"]} onSubmit={handleFormSubmit}>
 				<Input
+					className={getValidClassNames(styles["input"])}
+					control={control}
+					errors={errors}
+					label="First name"
+					name="firstName"
+					placeholder="First name"
+					type="text"
+					wrapperClassName={getValidClassNames(styles["input-wrapper"])}
```

Comment:
No need to use `getValidClassNames` if you have only one class to apply without condition
```suggestion
					wrapperClassName={styles["input-wrapper"]}
```